Steam Desktop Authenticator/ConfirmationFormWeb.cs

```diff
@@ -64,12 +64,16 @@ private async Task LoadData()
                             e.Graphics.FillRectangle(brush, panel.ClientRectangle);
                         }
                     };
-                    
+
                     if (!string.IsNullOrEmpty(confirmation.Icon))
                     {
-                       PictureBox pictureBox = new PictureBox() { Width = 60, Height = 60, Location = new Point(20, 20), SizeMode = PictureBoxSizeMode.Zoom };
-                       pictureBox.Load(confirmation.Icon);
-                       panel.Controls.Add(pictureBox);
+                        PictureBox pictureBox = new PictureBox() { Width = 60, Height = 60, Location = new Point(20, 20), SizeMode = PictureBoxSizeMode.Zoom };
+                        try
+                        {
+                            pictureBox.Load(confirmation.Icon);
+                        }
+                        catch { }

```

Comment:
Let's at least log the error in the catch block.

Reply:
Is there an example of logging code somewhere?

I think I can just add a Label with 404 text on top of the PictureBox.

Reply:
A simple `Console.WriteLine` call with the exception included in the message is sufficient here. No need to make it any more complicated.

Reply:
Done.

But I still added a visual representation of the error:
![image](https://github.com/Jessecar96/SteamDesktopAuthenticator/assets/7782218/39b4aae1-96f2-4ca6-b252-eed3e1cee164)